### PR TITLE
205: Ingesting Data merge

### PR DESCRIPTION
**Pull request by SinhaPratik**
Fixes | Relates to **[issue link]**

Content changes:

- Added L4 content for Ingesting Data at [205: Ingesting data](https://github.com/ibm-build-lab/VAD-VAR-Workshop/blob/Pratik/content/Watsonx/WatsonxData/205.md)
- Added images for the above at https://github.com/ibm-build-lab/VAD-VAR-Workshop/tree/Pratik/content/Watsonx/WatsonxData/images/205

Code changes:

- Any code changes here

### CODE REVIEW

File: content/Watsonx/WatsonxData/205.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
+---
+title: '205: Ingesting Data'
+timeToComplete: 20
+updated: 2024-02-02
+---
+
+## Ingesting Data
```
Comment (r2raman) [inline content/Watsonx/WatsonxData/205.md:7]: Add quiz alert tag 

<QuizAlert text='Heads Up! Quiz material will be flagged like this!' />

File: content/Watsonx/WatsonxData/205.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+This lab uses the ingest tool (ibm-lh) alongside the IBM watsonx.data Developer Edition that is 
+running in this lab. The Ingest tool is a separate install and currently needs to be downloaded 
+after IBM watsonx.data is started. The lab image (Developer Edition) contains a copy of this code 
+so you will not need to download it.
+
+
+In this section you will connect to your watsonx.data environment. The watsonx.data server 
+automatically starts all services. To check the status of the server, run the following commands
+using the instructions provided in **section 4.3. Command Line Access** of the “Prerequisites &
```
Comment (r2raman) [inline content/Watsonx/WatsonxData/205.md:29]: Provide the reference of the lab in the VEST site, not from the original content. In this case, you can refer to Lab 1: Environment Setup. Also, remove section 4.3.

**Review by r2raman — CHANGES_REQUESTED** [review]
Please work on the review comment

File: content/Watsonx/WatsonxData/205.md (hunk 1 above)
Comment (SinhaPratik) [reply content/Watsonx/WatsonxData/205.md:7]: Added

File: content/Watsonx/WatsonxData/205.md (hunk 2 above)
Comment (SinhaPratik) [reply content/Watsonx/WatsonxData/205.md:29]: Changed

**Review by r2raman — APPROVED** [review]

**Pull request merged**